src/integration-tests/bash/run.sh

```diff
@@ -2698,6 +2703,11 @@ function test_suite {
       # create domain6 in the default namespace with pvReclaimPolicy="Recycle", and verify that the PV is deleted once the domain and PVC are deleted
       test_create_domain_pv_reclaim_policy_recycle domain6
 
+      if [ "$JENKINS" = "true" ]; then
+        # create domain7 in the default namespace with weblogicDomainStorageType="NFS", and verify it
+        test_domain_creation domain7
```

Comment:
Adding a new domain lengthens the test time.   Instead, please piggyback on an existing domain - domain5 or domain6 look like they can be used.   

Also, add a note such as "# on Jenkins, this domain will also test NFS instead of HOSTPATH PV storage (search for [ "$DOMAIN_UID" == "domain5" ]).

Finally, modify the DOMAIN_UID check  to also check if on JENKINS...  && [ "$JENKINS" = "true" ]